alfred-telemetry-solr/build.gradle

```diff
@@ -0,0 +1,73 @@
+plugins {
+    id 'eu.xenit.docker' version '5.0.7' apply false
+    id 'java'
+    id 'idea'
+}
+
+subprojects {
+    repositories {
+	mavenCentral()
+	jcenter()
+	
+	maven {
+            url 'https://artifacts.alfresco.com/nexus/content/groups/private'
+            credentials {
+		username property("org.alfresco.maven.nexus.username")
+		password property("org.alfresco.maven.nexus.password")
+            }
+	}
+    }
+    
+    apply plugin: 'java'
+    apply from: "${project.projectDir}/overload.gradle"
+    apply plugin: 'eu.xenit.docker'
+
+    compileJava {
+        if ("solr4".equals(solrFlavor)) {
+            sourceCompatibility = '1.8'
+            targetCompatibility = '1.8'
+        } else {
+            sourceCompatibility = '11'
+            targetCompatibility = '11'
+        }
+    }
+
+
+    dependencies {
+        files(jar)
```

Comment:
why is this needed ?

Reply:
Comes from a previous version when solr images were is a separate subproject than the artifacts.
I removed it, thanks for reporting.